mithril-signer/src/runtime/runner.rs

```diff
@@ -255,9 +264,23 @@ impl Runner for SignerRunner {
             return Ok(false);
         }
 
-        if let Some(signer) =
-            pending_certificate.get_signer(self.services.single_signer.get_party_id())
-        {
+        // TODO XXX How handle errors ? It occurs in test_create_immutable_files_full_single_signature integration test (no signers in epoch 1)
+        // TODO XXX Do we warn and return "can not sign" or return an error ?
+        let current_signer_with_stake: Option<SignerWithStake> = {
```

Comment:
Maybe we only need a `current_signer: Option<Signer>`?

Reply:
Done